tabular/src/autogluon/tabular/models/abstract/model_trial.py

```diff
@@ -24,33 +25,40 @@ def model_trial(args, reporter: LocalStatusReporter):
     try:
         model, args, util_args = prepare_inputs(args=args)
 
+        epochs = util_args.pop("epochs")
+
+        report_probs = util_args.pop('report_probs')
+
         X_train, y_train = load_pkl.load(util_args.directory + util_args.dataset_train_filename)
         X_val, y_val = load_pkl.load(util_args.directory + util_args.dataset_val_filename)
 
         fit_model_args = dict(X_train=X_train, y_train=y_train, X_val=X_val, y_val=y_val)
         predict_proba_args = dict(X=X_val)
-        model = fit_and_save_model(model=model, params=args, fit_args=fit_model_args, predict_proba_args=predict_proba_args, y_val=y_val,
-                                   time_start=util_args.time_start, time_limit=util_args.get('time_limit', None), reporter=None)
+        model = fit_and_save_model(model=model, params=args, fit_args=fit_model_args, predict_proba_args=predict_proba_args,
+                                   y_val=y_val, time_start=util_args.time_start, time_limit=util_args.get('time_limit', None),
```

Comment:
.get('time_limit') instead of .get('time_limit', None)

Reply:
And why was reporter=None before?

Reply:
It was none because for other non-hyperband schedulers, there was no need to report back anything during the training. Updated the get("time_limit")